src/main/webapp/partials/instructorHelpSearch.jsp

```diff
@@ -0,0 +1,365 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
+<h4>
+    <a name="search">Search</a>
+</h4>
+<div id="contentHolder">
+    <br>
+    <ol style="list-style-type: none;">
+        <li>
+            <span class="text-bold">
+                <a name="searchStudents">
+                    <h3>1. Searching for students</h3>
+                </a>
+            </span>
+            <div>
+                You can search for students by clicking on
+                <b>'Search'</b> on the top nav bar. You should see a search bar similar to the one below -
```

Comment:
Just like for the profiles section, replace the hyphen with a colon for this and all the other new sections too.